packages/terra-form-input/tests/nightwatch/InputField.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import InputField from '../../lib/InputField';
+
+const InputFieldExample = () => (
```

Comment:
Can we update this test example to togglable between valid & invalid state for screenshot testing?